src/components/Navigation/styles.scss

```
@@ -0,0 +1,88 @@
+// Mobile portrait
+.wrapper {
+	width: 100%;
+	height: 100%;
+	padding-top: 10vh;
+	background-color: red;
+}
+
+	.itemContainer {
+		display: flex;
+		width: 100%;
+		height: 75vh;
```

```suggestion
		height: 75%;
```
Try not to use solid units like `vh`, `px` etc when you've used relative units in the parent - we've already set ourselves up to use `%` in the above, and if we use `vh` down here then it has the opportunity to break - for example, we want the component to adapt if the `.wrapper` is set to `height: 30vh` for example - we'd want everything inside it to adjust to accomodate that, whereas this would break it.